src/CSS/Calculator.css

```diff
@@ -102,9 +102,8 @@
 }
 
 ul {
-  list-style-type: none;
+  list-style-type: disc;
```

Comment:
idea for a future task: since we now use styled components more than CSS files, we should probably migrate all of our CSS to JS.

Reply:
Yeah I have been thinking about this one. There are a small pile of clean up changes I want to start working on next it will definitely be in there.